.github/workflows/tsdoc.yml

```
@@ -0,0 +1,73 @@
+
+    # Specify runner + deployment step
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)